Build failed on Fedora 25 32 bit due to using %zu for long unsigned int

>   136s] rsa.c: In function 'rsa_generate_keypair':
[  136s] rsa.c:328:51: error: format '%zu' expects argument of type 'size_t', but argument 3 has type 'long unsigned int' [-Werror=format=]
[  136s]          fprintf(stderr, "Unexpected RSA e value %zu, key generation failed\n", e);
[  136s]                                                    ^
`


PR with the fix will follow soon

--- a/src/lib/rsa.c
+++ b/src/lib/rsa.c
@@ -325,7 +325,7 @@ rsa_generate_keypair(pgp_key_t *         keydata,
     }
 
     if (e != 65537) {
-        fprintf(stderr, "Unexpected RSA e value %zu, key generation failed\n", e);
+        fprintf(stderr, "Unexpected RSA e value %lu, key generation failed\n", e);
         return false;
     }
 

